parserasgeo/features/cross_section.py

```diff
@@ -428,6 +428,12 @@ def import_geo(self, line, geo_file):
 
     def __str__(self):
         return 'Bank Sta=' + str(self.left) + ',' + str(self.right) + '\n'
+    
+class BankStationError(Exception):
+    """
+    An error if the bank stations are not correct
```

Comment:
Can you expand on how and why the bank stations might be wrong? e.g. is this if bank stations are corrupt in the geo file or is this exception raised if the user/calling code creates bank stations that don't have a corresponding point in the cross section geometry?